addons/payment/static/src/js/express_checkout_form.js

```diff
@@ -90,6 +90,12 @@ publicWidget.registry.PaymentExpressCheckoutForm = publicWidget.Widget.extend({
     _updateAmount(newAmount, newMinorAmount) {
         this.txContext.amount = parseFloat(newAmount);
         this.txContext.minorAmount = parseInt(newMinorAmount);
+        const expressCheckoutContainer = document.querySelector(
+                'div[name="o_express_checkout_container"]'
```

Comment:
The fix looks good to me but we should use existing helper methods when they exist. In this case, `_getExpressCheckoutForms` abstracts away the name of the divs to look for and handles multiple payment forms on the same page.

Reply:
This seems to not be working for this case.